sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/parquet/ParquetIOIT.java

```diff
@@ -137,6 +140,11 @@ public void writeThenReadAll() {
             .apply(
                 "Gather read start time",
                 ParDo.of(new TimeMonitor<>(PARQUET_NAMESPACE, "readStart")))
+            .apply(
+                "Collect byte count", ParDo.of(new ByteMonitor<>(PARQUET_NAMESPACE, "readBytes")))
```

Comment:
Actually, why collect both `writeBytes` and `readBytes`? is not `writeBytes` enough? After all, if the PAssert says that the collections are equal then that means both read and write PCollections are the same, right? Is that the reason why you skipped write/read bytes separation in AvroIOIT?

Probably name it simply `byteCount` as in `AvroIOIT`

Reply:
Changed the name and removed the extra byte counting. Thanks for the suggestion